GUI/Main/MainTime.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+using System.Threading.Tasks;
+using System.Linq;
+using CKAN.Extensions;
+using CKAN.Versioning;
+
+namespace CKAN
+{
+    public partial class Main
+    {
+        private void playTimeMenuItem_Click(object sender, EventArgs e)
+        {
+            // Run in a background task so GUI thread can react to user
+            Task.Factory.StartNew(() => playTime());
```

Comment:
This should probably run on the GUI thread because it manipulates the GUI (adds items to a list view).